pydantic_settings/sources.py

```diff
@@ -23,6 +23,9 @@
 
 DotenvType = Union[Path, List[Path], Tuple[Path, ...]]
 
+#
+ENV_FILE_SENTINEL: DotenvType = Path('')
```

Comment:
You said you added some comments above this — but it looks like they haven't made their way in? I just see `#` lol